### PR TITLE
Release 4.1.3

### PR DESCRIPTION
**Pull request by johnhwhite**
- Replace theme conditionals in templates with new skyIfTheme directive #52

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-avatar?_sp.spa=pr-release-4.1.3](https://developer.blackbaud.com/skyux-avatar?_sp.spa=pr-release-4.1.3)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# 4.1.3 (2020-03-16)
+
+- Replaced theme conditionals in templates with new skyIfTheme directive. [#52](https://github.com/blackbaud/skyux-avatar/pull/52)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md]:
```suggestion
- Replaced theme conditionals in templates with the new `skyIfTheme` directive. [#52](https://github.com/blackbaud/skyux-avatar/pull/52)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# 4.1.3 (2020-03-16)
```
Comment (blackbaud-johnly) [inline CHANGELOG.md]:
```suggestion
# 4.1.3 (2020-03-17)
```

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-avatar?_sp.spa=pr-release-4.1.3](https://developer.blackbaud.com/skyux-avatar?_sp.spa=pr-release-4.1.3)

**Review by blackbaud-johnly — APPROVED** [review]

**Pull request merged**